staging/cse/windows/azurecnifunc.tests.ps1

```diff
@@ -6,7 +6,7 @@ BeforeAll {
 Describe 'GetBroadestRangesForEachAddress' {
 
     It "Values '<Values>' should return '<Expected>'" -TestCases @(
-        @{ Values = @('10.240.0.0/12', '10.0.0.0/8'); Expected = @('10.0.0.0/8', '10.240.0.0/12')}
+        @{ Values = @('10.240.0.0/12', '10.0.0.0/8'); Expected = @('10.240.0.0/12', '10.0.0.0/8')}

```

Comment:
Will current ut fail without these changes?

Reply:
Without "ordered", we may need to change the order of unit test cases. It's a bit strange that when I test on my computer, the unit test passed. But the returned values have the opposite order compared with expected values on ut pipeline.

Reply:
Sure. Investigating.

Reply:
The problem is caused by the version of Powershell. Fix it by changing "pwsh" to "powershell" in validate-windows-ut.yml.